extra-wm/i3status-rust/autobuild/build

```diff
@@ -0,0 +1,7 @@
+cargo build --release --locked
```

Comment:
Please annotate your build script with `abinfo`, as required by the [AOSC OS Package Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/#scripting).